app/uk/gov/nationalarchives/omega/editorial/models/package.scala

```diff
@@ -42,11 +42,24 @@ package object editSetRecords {
         |  "endDateYear" : "1962",
         |  "legalStatus": "ref.1",
         |  "placeOfDeposit" : "1",
-        |  "note": "A note about COAL.2022.V1RJW.P."
+        |  "note": "A note about COAL.2022.V1RJW.P.",
+        |  "relatedMaterial" : [
+        |    {
+        |      "description" : "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths."
+        |    },
+        |    {
+        |      "linkHref" : "#;",
+        |      "linkText" : "COAL 80/80/3"
+        |    },
+        |    {
+        |      "linkHref" : "#;",
+        |      "linkText" : "COAL 80/80/2",
+        |      "description" : "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths."
+        |    }
+        |  ]
         |} """.stripMargin
     )
-    .validate[EditSetRecord]
-    .get
+    .as[EditSetRecord]
```

Comment:
Why no 'validate' here and with the second, but in all other cases?

Reply:
I changed these json objects to use `.as[...]` instead of `.validate[...]` because `as` throws the json parse exception, whereas validate and then get gives you a non-descriptive exception with no parse error, so it's harder to see what failed.

The ones that still use `validate` got merged in from main, I will change them to use as.

Reply:
Sounds like `as` is the way to go with all of them. Good.

Confirmed (all).